apps/quorum/src/server.rs

```diff
@@ -66,7 +79,17 @@ lazy_static! {
     /// - The `Receiver` side of a watch channel performs the detection based on if the change
     ///   happened after the subscribe, so it means all listeners should always be notified
     ///   correctly.
-    pub static ref EXIT: watch::Sender<bool> = watch::channel(false).0;
+
+    // Make sure at least one receiver is created
```

Comment:
this seems a bit hacky. can't we call `send_modify` here instead to make sure value changes even without receivers?